### PR TITLE
GAM SS doc, add step to update Prebid config

### PR DESCRIPTION
**Pull request by genwhittTTD**
- In GAM SS doc, add info re updating Prebid config.
- In both Prebid docs, add an "optional" section linking back for the GAM SS steps.

### CODE REVIEW

File: docs/guides/google-ss-integration.md [hunk 1]
```diff
@@ -31,14 +31,36 @@ With this framework, the following steps occur:
...
 
 :::important
 When you're following the steps, in [Select allowed secure signals](https://support.google.com/admanager/answer/10488752#select-signals), under **Web Signal Deploy Option**, choose **Google Deploy**.
-If you choose the **Prebid User ID Module**, your UID2s will not be correctly processed unless you also choose the **Use your Prebid configuration to automatically configure your Secure signals settings** field.
-Before saving your configuration, double-check that you've selected the correct option.
+If you choose the **Prebid User ID Module**, your UID2s will not be correctly processed unless you also choose the **Use your Prebid configuration to automatically configure your Secure signals settings** field. Before saving your configuration, double-check that you've selected the correct option.
+
+If you're using Prebid, you must also update your Prebid.js configuration: see [Optional: Enable Secure Signals in a Third-Party Bidder](#optional-enable-secure-signals-in-a-third-party-bidder).
```
Comment (sunnywu) [inline docs/guides/google-ss-integration.md:40]: _If you're using **Prebid.js**_

File: docs/guides/google-ss-integration.md [hunk 2]
```diff
@@ -31,14 +31,36 @@ With this framework, the following steps occur:
...
+For details, see [Share encrypted signals with bidders](https://support.google.com/admanager/answer/10488752) in the Google documentation, and then follow the steps in [Use a third-party signal provider](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/securesignals) to switch on UID2 as your signal provider.
 
 :::important
 When you're following the steps, in [Select allowed secure signals](https://support.google.com/admanager/answer/10488752#select-signals), under **Web Signal Deploy Option**, choose **Google Deploy**.
-If you choose the **Prebid User ID Module**, your UID2s will not be correctly processed unless you also choose the **Use your Prebid configuration to automatically configure your Secure signals settings** field.
-Before saving your configuration, double-check that you've selected the correct option.
+If you choose the **Prebid User ID Module**, your UID2s will not be correctly processed unless you also choose the **Use your Prebid configuration to automatically configure your Secure signals settings** field. Before saving your configuration, double-check that you've selected the correct option.
```
Comment (sunnywu) [inline docs/guides/google-ss-integration.md:38]: Could we extract line 38 - 63 and put it all under a section _Optional: Enable Secure Signals in a Third-Party Bidder via Prebid.js_

_your UID2s will not be correctly processed unless you also choose the **Use your Prebid configuration to automatically configure your Secure signals settings** field. Before saving your configuration, double-check that you've selected the correct option.  

You'll also need to update the configuration in the third-party bidder - update the `encryptedSignalSources` section in your Prebid configuration, as shown in the following code:_


Too many "if you are using prebid.js...." - best to group them all in a single section

File: docs/guides/google-ss-integration.md [hunk 3]
```diff
@@ -31,14 +31,36 @@ With this framework, the following steps occur:
...
+
+If you're using Prebid, you must also update your Prebid.js configuration: see [Optional: Enable Secure Signals in a Third-Party Bidder](#optional-enable-secure-signals-in-a-third-party-bidder).
 :::
 
+### Optional: Enable Secure Signals in a Third-Party Bidder
+
+If you want to enable Secure Signals support in a third-party bidder such as Prebid.js, after updating the Secure Signals settings you'll also need to update the configuration in the third-party bidder.
```
Comment (sunnywu) [inline docs/guides/google-ss-integration.md:45]: Prebid.js is a _third-party signal provider_ , not _third-party bidder_

File: docs/guides/google-ss-integration.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/guides/google-ss-integration.md:40]: Done.

File: docs/guides/google-ss-integration.md (hunk 3 above)
Comment (genwhittTTD) [reply docs/guides/google-ss-integration.md:45]: This mod is made redundant by the mods made for the previous section. Also we don't use above and below in docs for accessibility reasons. See mods for previous comment.

File: docs/guides/google-ss-integration.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/guides/google-ss-integration.md]: Updated... thx.

**Review by sunnywu — APPROVED** [review]

**Pull request merged**